mmv1/third_party/terraform/website/docs/r/cloudfunctions_function.html.markdown

```diff
@@ -146,7 +146,12 @@ Eg. `"nodejs10"`, `"nodejs12"`, `"nodejs14"`, `"python37"`, `"python38"`, `"pyth
 * `source_archive_object` - (Optional) The source archive object (file) in archive bucket.
 
 * `source_repository` - (Optional) Represents parameters related to source repository where a function is hosted.
-  Cannot be set alongside `source_archive_bucket` or `source_archive_object`. Structure is [documented below](#nested_source_repository).
+  Cannot be set alongside `source_archive_bucket` or `source_archive_object`. Structure is [documented below](#nested_source_repository). It must match the pattern `projects/{project}/locations/{location}/repositories/{repository}`.* 
+
+* `docker_repository` - (Optional) User managed repository created in Artifact Registry optionally with a customer managed encryption key. If specified, deployments will use Artifact Registry. This is the repository to which the function docker image will be pushed after it is built by Cloud Build. If unspecified, Container Registry will be used by default, unless specified otherwise by other means, e.g. `docker_registry`. [Beta] While the GCF CMEK is in GA, the Terraform support for Artifact Registry that the GCF CMEK relies on is still in beta.
```

Comment:
I had suggested adding the "Beta" section because I thought the functionality for this field was intended to be different between GA and beta - but actually, the field is the same; it's just that it can only be used by providing a resource that's beta. 

The reference to docker_registry should also be removed since the field doesn't exist yet.

```suggestion
* `docker_repository` - (Optional) User managed repository created in Artifact Registry optionally with a customer managed encryption key. If specified, deployments will use Artifact Registry. This is the repository to which the function docker image will be pushed after it is built by Cloud Build. If unspecified, Container Registry will be used by default.
```

Reply:
Got it, updated the doc.